src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -1804,6 +1804,7 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST(GUITest_regression_scenarios::test_7451);
     REGISTER_TEST(GUITest_regression_scenarios::test_7456);
     REGISTER_TEST(GUITest_regression_scenarios::test_7460);
+    REGISTER_TEST_T(GUITest_regression_scenarios::test_7465, minutes(14));
```

Comment:
If the task fails with out-of-memory error why do you need such a big timeout?

Reply:
Opening this file is the most time consuming operation. After that Muscle task measure memory cnsumption.

Reply:
We can optimize opening of the file, because the opening itself is out of the test scope.
How to optimize it:
1) Use a FASTA file. Such files have a very good parser
or
2) Use UGENEDB file. Such  files are opened instantly.
or
3) Generate a FASTA file and open it as an alignment.

Reply:
1st option choosen
2nd is impossible - ugene can't convert msa object to ugenedb